app/views/case_workers/admin/allocations/_filter_tasks.html.haml

```diff
@@ -1,8 +1,8 @@
 .form-group#filter-tasks.dtFilter.dtFilterAGFS{:data => {scheme: 'agfs'}}
   %fieldset.inline
-    %label.form-label{:for => "filterAGFS"}
+    %label.form-label-bold{:for => "filterAGFS"}
       = t('.case_types')
-    %select#filterAGFS.form-control{:name => "tasks"}
+    %select#filterAGFS.form-control{:name => "tasks", :style => "width:100%"}
```

Comment:
Inline styles?! 🤔

Reply:
I want to go over these views again at some point. will raise a ticket.

Reply:
https://dsdmoj.atlassian.net/browse/CBO-657